tests/testing.py

```diff
@@ -99,6 +102,8 @@ def script(self, *args):
             'FIPS_BOOT_CFG_DIR': str(self.etc_dir),
```

Comment:
I think you missed a REPO_URL setting -> ESM_REPO_URL in testing.py

Reply:
Fixed. Sorry I had to force push due to other reasons, but you can see the specific commit that fixed this (3d189615f3ebf705c8577be82d8a23027e686c73)